frontend-web/webclient/app/Applications/ApplicationsRow.tsx

```diff
@@ -92,74 +130,35 @@ const ApplicationRow: React.FunctionComponent<ApplicationRowProps> = ({
-                                    title={app.title}
-                                    description={app.description}
-                                    logo={app.id.toString()}
-                                    logoType="GROUP"
-                                    isFavorite={false}
-                                />
-                            </Link>
-                        )}
-                    </Flex>
-                </div>
-            :
-                scrolling ?
-                    <div ref={scrollRef} className={ApplicationRowContainerClass}>
-                        <Grid
-                            gridGap="25px"
-                            gridTemplateRows={"repeat(1, 1fr)"}
-                            gridTemplateColumns={"repeat(auto-fill, 166px)"}
-                            style={{gridAutoFlow: "column"}}
-                        >
-                            {items.map(app =>
-                                <>
-                                    <Link key={app.id} to={groupCardLink(app)}>
-                                        <AppCard
-                                            type={ApplicationCardType.TALL}
-                                            title={app.title}
-                                            description={app.description}
-                                            logo={app.id.toString()}
-                                            logoType="GROUP"
-                                            isFavorite={false}
-                                        />
-                                    </Link>
-                                </>
-                            )}
-                        </Grid>
-                    </div>
-                :
-                    <div ref={scrollRef} className={ApplicationRowContainerClass}>
-                        <Flex
-                            justifyContent="space-evenly"
-                            gap="10px"
-                            py="10px"
-                        >
-                            {items.map(app =>
-                                <Link key={app.id} to={groupCardLink(app)}>
-                                    <AppCard
-                                        type={ApplicationCardType.TALL}
-                                        title={app.title}
-                                        description={app.description}
-                                        logo={app.id.toString()}
-                                        logoType="GROUP"
-                                        isFavorite={false}
-                                    />
-                                </Link>
-                            )}
-                        </Flex>
-                    </div>
-            }
+            <div
+                ref={scrollRef}
+                className={ApplicationRowContainerClass}
+                data-space-between={style === AppCardStyle.WIDE ? (items.length > 3) : (items.length > 6)}
```

Comment:
`style` is overloaded for me. Calling it `cardStyle` would be better, so it doesn't seem like the `style` is an object of CSSProperties.